UI/FleetButton.cpp

```diff
@@ -172,13 +172,28 @@ FleetButton::FleetButton(const std::vector<int>& fleet_IDs, SizeType size_type)
 
     // select icon(s) for fleet(s)
     int num_ships = 0;
+    m_fleet_blockaded = false;
     for (std::shared_ptr<const Fleet> fleet : fleets) {
-        if (fleet)
+        if (fleet) {
             num_ships += fleet->NumShips();
+            if (fleet->Blockaded())
+                m_fleet_blockaded = true;
+        }
+    }  
+
+    // add RotatingGraphics for all icons needed
+    if (m_fleet_blockaded) {
+        std::shared_ptr<GG::Texture> blockaded_texture = FleetBlockadedIcon(size_type);
+        RotatingGraphic* icon = new RotatingGraphic(blockaded_texture, GG::GRAPHIC_FITGRAPHIC | GG::GRAPHIC_PROPSCALE);
+        icon->SetRPM(0.0f);
```

Comment:
if it's not rotating, why do you use a `RotatingGraphic` instead of a `StaticGraphic` ?